client/src/components/CommentCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Card from 'react-bootstrap/Card';
+import {AiFillEdit } from 'react-icons/ai';
+import {MdDelete } from 'react-icons/md';
+import EditCommentPopup from './EditCommentPopUp';
+import StarDisplay from './StarDisplay';
+
+export default function CommentCard({ text, userName, date, rating, icon, commentId, commentUpdated }) {
+
+  const [show, setShow] = useState(false)
+
+  async function deleteComment(id) {
+
+    try {
+        const deletePost = await fetch(`/api/comment/${id}`, 
+        {method: "DELETE"})
+    } catch (error) {
+        console.error(error.message)
+    }
+
+    if(commentUpdated !== undefined) commentUpdated();
```

Comment:
It looks like `commentUpdated` is a callback passed via props; here's a few things you should do:

1. Change the name to `onCommentUpdated`. Starting callbacks with the word `on` is the React convention; think about a `<button>` or `<input>`, they're `onClick` or `onChange`
2. You are checking to see if this callback is defined. A simpler way of writing that is by using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
   ```js
   commentUpdated?.();
   ```